src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -33,6 +62,7 @@ export const TodoFilter: React.FC = () => {
             data-cy="clearSearchButton"
             type="button"
             className="delete"
+            onClick={clearQuery}
```

Comment:
this button shouldn't be visible if no text typed in the input - check working version